relay/polkadot/constants/src/lib.rs

```diff
@@ -109,6 +109,7 @@ pub mod xcm {
 		// The bodies corresponding to the Polkadot OpenGov Origins.
 		pub const FELLOWSHIP_ADMIN_INDEX: u32 = 1;
 		// The body corresponding to the Treasurer OpenGov track.
+		#[deprecated = "use `xcm::latest::BodyId::Treasury` instead"]
```

Comment:
Can you please add a removal date? We are trying to that in the SDK at least like this: https://github.com/paritytech/polkadot-sdk/blob/a5370fb1877cc9672cb47fea36a27cfed3f4da63/substrate/frame/support/src/storage/migration.rs#L40

Reply:
done